SeaPublicWebsite.UnitTests/EpcTests.cs

```diff
@@ -0,0 +1,1377 @@
+            {
+                AssessmentType = "RdSAP",
+                WindowsDescription = new List<string>
+                {
+                    "single glazed"
+                }
+            },
+            new Epc
+            {
+                GlazingType = GlazingType.SingleGlazed
+            }),
+        new(
+            "Can parse glazing type both (some)",
+            new EpbEpcAssessmentDto
+            {
+                AssessmentType = "RdSAP",
+                WindowsDescription = new List<string>
+                {
+                    "some double glazing"
+                }
+            },
+            new Epc
+            {
+                GlazingType = GlazingType.Both
+            }),
+        new(
+            "Can parse glazing type both (partial)",
+            new EpbEpcAssessmentDto
+            {
+                AssessmentType = "RdSAP",
+                WindowsDescription = new List<string>
+                {
+                    "partial triple glazing"
+                }
+            },
+            new Epc
+            {
+                GlazingType = GlazingType.Both
+            }),
+        new(
+            "Can parse glazing type both (mostly)",
+            new EpbEpcAssessmentDto
+            {
+                AssessmentType = "RdSAP",
+                WindowsDescription = new List<string>
+                {
+                    "mostly multiple glazing"
+                }
+            },
+            new Epc
+            {
+                GlazingType = GlazingType.Both
+            }),
+        new(
+            "Can parse glazing type both (throughout)",
+            new EpbEpcAssessmentDto
+            {
+                AssessmentType = "RdSAP",
+                WindowsDescription = new List<string>
+                {
+                    "multiple glazing throughout"
+                }
+            },
+            new Epc
+            {
+                GlazingType = GlazingType.Both
+            }),
+        new(
+            "Can parse glazing type double or triple (full)",
+            new EpbEpcAssessmentDto
+            {
+                AssessmentType = "RdSAP",
+                WindowsDescription = new List<string>
+                {
+                    "fully double glazed"
+                }
+            },
+            new Epc
+            {
+                GlazingType = GlazingType.DoubleOrTripleGlazed
+            }),
+        new(
+            "Can parse glazing type double or triple (high)",
+            new EpbEpcAssessmentDto
+            {
+                AssessmentType = "RdSAP",
+                WindowsDescription = new List<string>
+                {
+                    "High performance glazing"
+                }
+            },
+            new Epc
+            {
+                GlazingType = GlazingType.DoubleOrTripleGlazed
+            })
+    };
+    
+    private static readonly EpcTestCase[] HeatingTypeTestCases =
+    {
+        new(
```

Comment:
For here and some other places I don't think you've covered all the options. How have you the options to cover?